pages/_app.tsx

```diff
@@ -0,0 +1,15 @@
+import "./index.scss";
```

Comment:
Note that global CSS is importable without the `module.scss` type in the `_*.tsx` files in the `pages` folder.